modules/op-difference-between-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="difference-between-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Difference between non-versioned and versioned cluster tasks
```

Comment:
This title is incomplete. Please make the first word plural or use a definite article.
 ```suggestion
= Differences between non-versioned and versioned cluster tasks
```
or 
```suggestion
= The difference between non-versioned and versioned cluster tasks
```